### PR TITLE
Login logout

### PR DESCRIPTION
**Pull request by HoangThiTuanDung**

### CODE REVIEW

File: app/Controller/UsersController.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+		if ($user['User']['is_admin']) {
+			$this->redirect(['action' => 'admin']);
+		}
+
+		$this->loadModel('Position');
+		$this->loadModel('Division');
+		$this->loadModel('Group');
+		$this->loadModel('Team');
+
+		$position_id = $user['User']['position_id'];
```
Comment (tranluan91) [inline app/Controller/UsersController.php]: Variable's name must is camelCase

File: app/View/Layouts/default.ctp [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+	?>
+	<link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link rel="stylesheet" href="//code.jquery.com/ui/1.11.3/themes/smoothness/jquery-ui.css">
+	<link rel="stylesheet" href="http://cdnjs.cloudflare.com/ajax/libs/bootstrap-select/1.6.3/css/bootstrap-select.css">
+	<script src="//code.jquery.com/jquery-1.10.2.js"></script>
+	<script src="//code.jquery.com/ui/1.11.3/jquery-ui.js"></script>
+	<script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
+	<script src="https://cloud.github.com/downloads/knockout/knockout/knockout-2.1.0.js"></script>
+	<script src="http://cdnjs.cloudflare.com/ajax/libs/bootstrap-select/1.6.3/js/bootstrap-select.js"></script>
```
Comment (tranluan91) [inline app/View/Layouts/default.ctp]: All of js or css libraries must setup in local.

File: app/View/Users/index.ctp [hunk 3]
```diff
@@ -0,0 +1,38 @@
+<div class="profile">
+	<div class="left" style="margin-right: 20px; float: left; display: inline-block">
```
Comment (tranluan91) [inline app/View/Users/index.ctp]: Don't write CSS inline. Please move it into css file

File: app/View/Users/login.ctp [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<h2>Login</h2>
+<?php
+	echo $this->Form->create('User', ['type' => 'post']);
+	echo $this->Form->input('email', ['label' => 'Email', 'type' => 'email', 'class' => 'form-control', 'placeholder' => 'Email']);
+	echo $this->Form->input('password', ['label' => 'Password', 'type' => 'password', 'class' => 'form-control', 'placeholder' => 'Password']);
```
Comment (tranluan91) [inline app/View/Users/login.ctp]: Please use ```__()``` to using i18n

File: app/View/Layouts/default.ctp [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+			<nav class="navbar navbar-default">
+				<div class="container-fluid">
+					<div class="navbar-header">
+						<a class="navbar-brand" href="/" class="active">FES Project</a>
+					</div>
+					<div>
+						<ul class="nav navbar-nav navbar-right">
+							<?php if (!$this->Session->read('Auth.User')): ?>
+								<li><a href="/login">Login</a></li>
+							<?php elseif ($this->Session->read('Auth.User.User.email') != 'admin@framgia.com'): ?>
```
Comment (tranluan91) [inline app/View/Layouts/default.ctp]: 1. You can get data of currently logged user:
    http://book.cakephp.org/2.0/en/core-libraries/components/authentication.html#AuthComponent::user
2. Why did you compare between userEmail and ```admin@framgia.com```? 
If you need do it, please move condition into controller

File: app/View/Layouts/default.ctp [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+		<div id="content">
+			<nav class="navbar navbar-default">
+				<div class="container-fluid">
+					<div class="navbar-header">
+						<a class="navbar-brand" href="/" class="active">FES Project</a>
+					</div>
+					<div>
+						<ul class="nav navbar-nav navbar-right">
+							<?php if (!$this->Session->read('Auth.User')): ?>
+								<li><a href="/login">Login</a></li>
```
Comment (tranluan91) [inline app/View/Layouts/default.ctp]: Please use HtmlHelper:
http://book.cakephp.org/2.0/en/core-libraries/helpers/html.html

File: app/View/Users/index.ctp [hunk 7]
```diff
@@ -0,0 +1,38 @@
+<div class="profile row">
+	<div class="col-xs-4">
+		<div class="thumbnail">
+			<?php echo $this->Html->image($user['User']['image'], ['width' => '300px', 'height' => '300px']); ?>
```
Comment (tranluan91) [inline app/View/Users/index.ctp]: Please move css into css file

File: app/Controller/AppController.php [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+class AppController extends Controller {
+	public $helpers = [
+		'Session'
+	];
+
+	var $components = [
+		'Auth' => [
+			'loginRedirect' => ['controller' => 'users', 'action' => 'index'],
+			'logoutRedirect' => ['controller' => 'users', 'action' => 'login'],
+			'authError' => 'You must be logged in to view this page.',
```
Comment (tranluan91) [inline app/Controller/AppController.php]: Please use ```__()```

File: app/Config/routes.php [hunk 9]
```diff
@@ -0,0 +1,180 @@
...
+	Router::connect(
+		'/admin/positions/index',
+		['controller' => 'positions', 'action' => 'index']
+	);
+	Router::connect(
+		'/admin/positions/create',
+		['controller' => 'positions', 'action' => 'create']
+	);
+	Router::connect(
+		'/admin/positions/update/*',
```
Comment (dinhhoanglong91) [inline app/Config/routes.php]: This task is about login and logout. Do you need this route for login/logout?
Please remove unused code.

File: app/Controller/UsersController.php [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+	public function index() {
+		$data = $this->Auth->user();
+		$user = $this->User->findById($data['id']);
+		if ($user['User']['is_admin']) {
+			$this->redirect(['action' => 'admin']);
+		}
+
+		$this->loadModel('Position');
+		$this->loadModel('Division');
+		$this->loadModel('Group');
```
Comment (dinhhoanglong91) [inline app/Controller/UsersController.php]: Where is model Group? Can't find Group.php here.
Maybe Group isn't needed for login/logout?
User belongs to Group, why don't you link model together use "belongsTo" or "hasMany"?

**Pull request merged**